src/workspace/package.json

```diff
@@ -39,5 +40,10 @@
     "dialogflow": "^1.2.0",
     "openfin-notifications": "^1.11.0",
     "rxjs": "^7.5.2"
+  },
```

Comment:
this is one of my left-over bits from the last time I was working with the generated `TradingGateway.ts` which was the same across client / workspace but formatted differently so it was quite difficult to establish that they were in sync!